akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -467,6 +470,14 @@ object Multipart {
           FastFuture.successful(this)
         override def productPrefix = "FormData.BodyPart.Strict"
       }
+
+      /** Java API */
+      object StrictBuilder {
+        def apply(_name: String, _entity: HttpEntity.Strict,
```

Comment:
A common pattern we do is to have static "factories" in `TheThing[s]` classes for javadsl.
For example `HttpEntity` and `HttpEntities.create`, we could do the same here, with `Multiparts`, put the factories there